### PR TITLE
[debops.gitlab_runner] Add ability to configure runner cache as shared

### PR DESCRIPTION
**Pull request by dmart**

### CODE REVIEW

File: ansible/roles/debops.gitlab_runner/defaults/main.yml [hunk 1]
```diff
@@ -665,8 +665,14 @@ gitlab_runner__cache_bucket_location: ''
 #
 # Enable or disable connections to the S3 service over HTTP instead of HTTPS.
 gitlab_runner__cache_insecure: False
+
```
Comment (bfabio) [inline ansible/roles/debops.gitlab_runner/defaults/main.yml]: Can you split this whitespace fix into a different PR?

**Comment by dmart** [comment]
removed

**Review by bfabio — APPROVED** [review]
LGTM.

**Review by drybjed — APPROVED** [review]
